addons/mail/models/mail_render_mixin.py

```diff
@@ -357,53 +357,48 @@ def _render_template_jinja(self, template_txt, model, res_ids,
         if any(r is None for r in res_ids):
             raise ValueError(_('Template rendering should be called on a valid record IDs.'))
 
-        # TDE note: support 'safe' context key as backward compatibility for 6dde919bb9850912f618b561cd2141bffe41340c
-        if options is None:
-            options = {}
-        no_autoescape = options.get('render_safe') or self._context.get('safe')
-
         results = dict.fromkeys(res_ids, u"")
         if not template_txt:
             return results
 
-        # try to load the template
-        try:
-            jinja_env = jinja_safe_template_env if no_autoescape else jinja_template_env
-            template = jinja_env.from_string(tools.ustr(template_txt))
-        except Exception:
-            _logger.info("Failed to load template %r", template_txt, exc_info=True)
-            return results
+        # todo: security detect if template is dynamic
```

Comment:
C'est fait dans un commit suivant ou bien c'est pour une autre PR ? Je mets le commentaire maintenant parce que si j'attends d'avoir vu les commits suivants je vais oublier de revenir ici.

Reply:
To cleanup and to check with @std-odoo